### PR TITLE
feat(cmd/paratime/list): Add column DENOMINATION(S)

### PR DESCRIPTION
**Pull request by amela**

### CODE REVIEW

File: cmd/paratime/list.go [hunk 1]
```diff
@@ -18,7 +21,7 @@ var listCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		cfg := cliConfig.Global()
 		table := table.New()
-		table.SetHeader([]string{"Network", "Paratime", "ID"})
+		table.SetHeader([]string{"Network", "Paratime", "ID", "DENOMINATION(S)"})
```
Comment (kostko) [inline cmd/paratime/list.go:24]:
```suggestion
		table.SetHeader([]string{"Network", "Paratime", "ID", "Denomination(s)"})
```
AFAIK capitalization happens during rendering.

File: cmd/paratime/list.go [hunk 2]
```diff
@@ -48,3 +52,19 @@ var listCmd = &cobra.Command{
...
+		// NativeDenominationKey = "_"
+		if nativeKey == "_" {
+			fmtDenom = fmt.Sprintf("%s[%d] (*)", denom.Symbol, denom.Decimals)
+		} else {
+			fmtDenom = fmt.Sprintf("%s[%d]", denom.Symbol, denom.Decimals)
+		}
+
+		fmtDenomArray = append(fmtDenomArray, fmtDenom)
+	}
+	return (strings.Join(fmtDenomArray, ", "))
```
Comment (kostko) [inline cmd/paratime/list.go:69]: Are newlines supported by the table formatter? Maybe that would look better.

Comment (amela) [reply cmd/paratime/list.go]: I think there is a problem with tests. On my computer, there is one space between the `NETWORK` and `PARATIME` columns, but in tests, there are multiple spaces between those two columns:

```diff --git a/examples/paratime/00-list.out b/examples/paratime/00-list.out
index fd93a16..0645e72 100644
--- a/examples/paratime/00-list.out
+++ b/examples/paratime/00-list.out
@@ -1,9 +1,9 @@
-NETWORK PARATIME        ID                                                                      DENOMINATION(S) 
-mainnet cipher          000000000000000000000000000000000000000000000000e199119c992377cb        ROSE[9] (*)    
-mainnet emerald         000000000000000000000000000000000000000000000000e2eaa99fc008f87f        ROSE[18] (*)   
-mainnet sapphire (*)    000000000000000000000000000000000000000000000000f80306c9858e7279        ROSE[18] (*)   
-testnet cipher          0000000000000000000000000000000000000000000000000000000000000000        TEST[9] (*)    
-testnet emerald         00000000000000000000000000000000000000000000000072c8[215](https://github.com/oasisprotocol/cli/actions/runs/9076794590/job/24940286293?pr=237#step:7:216)e60d5bca7        TEST[18] (*)   
-testnet pontusx         0000000000000000000000000000000000000000000000004febe52eb412b421        TEST[18]       
-                                                                                                EUROe[18] (*)  
-testnet sapphire (*)    000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c        TEST[18] (*)
+NETWORK	PARATIME    	ID                                                              	DENOMINATION(S) 
+mainnet	cipher      	000000000000000000000000000000000000000000000000e199119c992377cb	ROSE[9] (*)    	
+mainnet	emerald     	000000000000000000000000000000000000000000000000e2eaa99fc008f87f	ROSE[18] (*)   	
+mainnet	sapphire (*)	000000000000000000000000000000000000000000000000f80306c9858e7279	ROSE[18] (*)   	
+testnet	cipher      	0000000000000000000000000000000000000000000000000000000000000000	TEST[9] (*)    	
+testnet	emerald     	00000000000000000000000000000000000000000000000072c8215e60d5bca7	TEST[18] (*)   	
+testnet	pontusx     	0000000000000000000000000000000000000000000000004febe52eb412b421	TEST[18]       	
+       	            	                                                                	EUROe[18] (*)```

Comment (amela) [reply cmd/paratime/list.go]: Never mind, I solved it. :)

File: cmd/paratime/list.go [hunk 3]
```diff
@@ -48,3 +52,22 @@ var listCmd = &cobra.Command{
...
+			fmtDenom = fmt.Sprintf("%s[%d] (*)", denom.Symbol, denom.Decimals)
+		} else {
+			fmtDenom = fmt.Sprintf("%s[%d]", denom.Symbol, denom.Decimals)
+		}
+
+		fmtDenomArray = append(fmtDenomArray, fmtDenom)
+	}
+	sort.Slice(fmtDenomArray, func(i, j int) bool {
+		return fmtDenomArray[i] < fmtDenomArray[j]
+	})
```
Comment (kostko) [inline cmd/paratime/list.go:71]:
```suggestion
	slices.Sort(fmtDenomArray)
```

File: cmd/paratime/list.go [hunk 4]
```diff
@@ -48,3 +52,22 @@ var listCmd = &cobra.Command{
...
+	return (strings.Join(fmtDenomArray, "\n"))
```
Comment (kostko) [inline cmd/paratime/list.go:72]:
```suggestion
	return strings.Join(fmtDenomArray, "\n")
```

File: cmd/paratime/list.go [hunk 5]
```diff
@@ -48,3 +52,22 @@ var listCmd = &cobra.Command{
 		table.Render()
 	},
 }
+
+func formatDenominations(denominations map[string]*config.DenominationInfo) string {
+	fmtDenomArray := []string{}
+	fmtDenom := ""
```
[the remark this answers is not in the document]
Comment (amela) [reply cmd/paratime/list.go]: linter does not like this
 

> Error: cmd/paratime/list.go:59:2: Consider pre-allocating `fmtDenomArray` (prealloc)
>   	var (
>   	^

[the remark this answers is not in the document]
Comment (kostko) [reply cmd/paratime/list.go]: Fine, then just do it for `fmtDenom` and preallocate like this (a small optimization to reduce allocations, likely irrelevant in this case):
```go
fmtDenomArray := make([]string, 0, len(denominations))
```

[the remark this answers is not in the document]
Comment (amela) [reply cmd/paratime/list.go]: Done

**Review by kostko — APPROVED** [review]

**Pull request merged**